idd/Energy+.idd.in

```diff
@@ -34955,6 +34955,4645 @@ ZoneHVAC:EvaporativeCoolerUnit,
+    A3, \field Availability Manager List Name
+        \note Enter the name of an AvailabilityManagerAssignmentList object.
+        \type object-list
+        \object-list SystemAvailabilityManagerLists
+    A4, \field Minimum Supply Air Temperature Schedule Name
+        \note Values in this schedule are used as a constraint in choosing the feasible settings for supply air flow rate and ouside air fraction in each operating mode. If this field is blank, no minimum is imposed.
+        \type object-list
+        \object-list ScheduleNames
+    A5, \field Maximum Supply Air Temperature Schedule Name
+        \note Values in this schedule are used as a constraint in choosing the feasible settings for supply air flow rate and outside air fraction in each operating mode. If this field is blank, no maximum is imposed.
+        \type object-list
+        \object-list ScheduleNames
+    A6, \field Minimum Supply Air Humidity Ratio Schedule Name
+        \note Values in this schedule are used as a constraint in choosing the feasible settings for supply air flow rate and outside air fraction in each operating mode. If this field is blank, no minimum is imposed.
+        \type object-list
+        \object-list ScheduleNames
+    A7, \field Maximum Supply Air Humidity Ratio Schedule Name
+        \note Values in this schedule are used as a constraint in choosing the feasible settings for supply air flow rate and outside air fraction in each operating mode. If this field is blank, no maximum is imposed.
+        \type object-list
+        \object-list ScheduleNames
+    A8, \field Method to Choose Controlled Inputs and Part Runtime Fraction
+        \note Select the method that will be used to choose operating mode(s), supply air flow rate(s), outside air fraction(s) and part runtime fraction(s) in each time step. "Automatic" = chooses controlled inputs and part runtime fraction(s) to minimize resource use within each time step while best satisfying requested sensible cooling, dehumidification and ventilation, and subject to constraints. "User Defined" = EMS will be used to choose controlled inputs and part runtime fraction(s) in each time step. If this field is blank, default to "Automatic".
+        \type choice
+        \key Automatic
+        \key User Defined
+        \default Automatic
+    A9, \field Return Air Node Name
+        \type node
+        \note Return air node for the hybrid unit must be a zone exhaust node.
+   A10, \field Outside Air Node Name
+        \type node
+        \note Outside air node for the hybrid unit must be an outdoor air node.
+   A11, \field Supply Air Node Name
+        \type node
+        \note Supply air node for the hybrid unit must be a zone air inlet node.
+   A12, \field Relief Node Name
+        \type node
+        \note Relief node for the hybrid unit must be a zone exhaust node, unless flow is being balanced elsewhere.
+    N1, \field System Maximum Supply Air Flow Rate
+        \type real
+        \minimum> 0
```

Comment:
Any field with a minimum> 0 and no default should be a \required-field, otherwise if it's left blank it will error out.